packages/server/src/fhir/operations/export.test.ts

```diff
@@ -68,15 +68,13 @@ describe('Export', () => {
 
     // Check the export status
     const contentLocation = new URL(initRes.headers['content-location']);
+    await waitForAsyncJob(initRes.headers['content-location'], app, accessToken);
 
-    let resBody: any;
-    await waitFor(async () => {
```

Comment:
moving to waitForAsyncJob to ensure all jobs are completed in export tests. Introducing to this pattern for maintaining these export tests will no longer introduce race conditions where a test ends, but a job might still be running in the background.